File: src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveRepository.java
package no.nav.fo.veilarbregistrering.oppgave;

import no.nav.fo.veilarbregistrering.bruker.AktorId;

import java.util.List;

public interface OppgaveRepository {

    long opprettOppgave(AktorId aktørId, OppgaveType oppgaveType, long oppgaveId);

    List<OppgaveImpl> hentOppgaverFor(AktorId aktørId);
}

File: src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveRepository.kt
package no.nav.fo.veilarbregistrering.oppgave

import no.nav.fo.veilarbregistrering.bruker.AktorId

interface OppgaveRepository {
    fun opprettOppgave(aktorId: AktorId, oppgaveType: OppgaveType, oppgaveId: Long): Long
    fun hentOppgaverFor(aktorId: AktorId): List<OppgaveImpl>
}